src/actions/Actions.ts

```diff
@@ -332,15 +339,30 @@ const mergeImages = (localImages: ImageData[], uploadedImages: ImageData[]): Ima
     return mergedImages;
 };
 
-const loadPostsFromFeeds = async (feeds: Feed[]): Promise<Post[]> => {
+const loadPostsFromFeeds = async (swarm: Swarm.ReadableApi, feeds: Feed[]): Promise<Post[]> => {
     const rssFeeds = feeds.filter(feed => !isPostFeedUrl(feed.url));
     const postFeeds = feeds.filter(feed => isPostFeedUrl(feed.url));
-
     const allPostsCombined = await Promise.all([
         RSSPostManager.loadPosts(rssFeeds) as Promise<PublicPost[]>,
-        loadPosts(postFeeds),
+        loadRecentPosts(swarm, postFeeds),
     ]);
 
     const allPosts = allPostsCombined[0].concat(allPostsCombined[1]);
     return allPosts;
 };
+
+const getSwarmStorageSyncer = (signFeedDigest: Swarm.FeedDigestSigner, feed: LocalFeed) => {
```

Comment:
It might be better to have this in a separate file

Reply:
I was thinking the same, but then I couldn't decide where would I put it. I wouldn't put it under `swarm` or `swarm-social` because I think of them as separate generic packages and this code is specific to the app.